src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -4,6 +4,7 @@
 
 import java.util.Collection;
 import java.util.HashSet;
+import java.util.LinkedHashSet;
```

Comment:
Interesting choice of data structure here!
Is there a reason behind using a LinkedHashSet over a regular HashSet?

Reply:
Interesting, seems like using a LinkedHashSet will take up more memory than a normal HashSet but still preserves the runtime benefits of a set

Reply:
Interesting, seems like using a LinkedHashSet will take up more memory than a normal HashSet but still preserves the runtime benefits of a set